FirebaseAuth/Tests/Unit/FIRGetOOBConfirmationCodeRequestTests.m

```diff
@@ -142,6 +142,36 @@
  */
 static NSString *const kDynamicLinkDomain = @"test.page.link";
 
+/** @var kCaptchaResponseKey
+    @brief The key for the "captchaResponse" value in the request.
+ */
+static NSString *const kCaptchaResponseKey = @"captchaResponse";
+
+/** @var kTestCaptchaResponse
+    @brief Fake captchaResponse for testing the request.
+ */
+static NSString *const kTestCaptchaResponse = @"captchaResponse";
```

Comment:
nit: Can you make the values different than the key for all 3 fields? here and in the other test files. Thanks!